flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/config/AutoscalerStandaloneOptions.java

```diff
@@ -38,6 +38,12 @@ private static ConfigOptions.OptionBuilder autoscalerStandaloneConfig(String key
                     .withDeprecatedKeys("scalingInterval")
                     .withDescription("The interval of autoscaler standalone control loop.");
 
+    public static final ConfigOption<Integer> CONTROL_LOOP_PARALLELISM =
+            autoscalerStandaloneConfig("control-loop.parallelism")
+                    .intType()
+                    .defaultValue(100)
```

Comment:
FYI, the default for the Kubernetes operator is 200 but those two do not have to be aligned.